src/sentry/api/endpoints/authenticator_index.py

```diff
@@ -13,14 +14,28 @@ def get(self, request):
 
         # Currently just expose u2f challenge, not sure if it's necessary to list all
         # authenticator interfaces that are enabled
+        # this is a weird endpoint, looks like its only being used in test_user_authenticators_index.py
```

Comment:
maybe it's used in `getsentry`?

Reply:
will look for it

Reply:
@maxiuyuan did you figure anything else out with this?

Reply:
resolved in chat